packages/salesforcedx-vscode-core/package.nls.ja.json

```diff
@@ -56,10 +56,10 @@
   "org_logout_all_text": "SFDX: すべての認証済み組織からログアウト",
   "org_logout_default_text": "SFDX: Log Out from Default Org",
   "org_open_default_scratch_org_text": "SFDX: デフォルトの組織を開く",
-  "override_conflicts_on_push_description": "保存時の force:source:push コマンド実行時に、常に --forceoverwrite オプションを使用するかどうかを指定します。",
+  "ignore_conflicts_on_push_description": "保存時の project:deploy:start コマンド実行時に、常に --ignore-conflicts オプションを使用するかどうかを指定します。",
   "project_generate_text": "SFDX: プロジェクトを作成",
   "project_generate_with_manifest_text": "SFDX: マニフェストファイルを使用してプロジェクトを作成",
-  "push_or_deploy_on_save_enabled_description": "ローカルのソースファイルを保存した際に、自動的に force:source:push コマンド (ソースが追跡される組織) または、force:source:deploy コマンド (ソースが追跡されない組織) を実行するかどうかを指定します。",
+  "push_or_deploy_on_save_enabled_description": "ローカルのソースファイルを保存した際に、自動的に project:deploy:start コマンド (ソースが追跡される組織) または、force:source:deploy コマンド (ソースが追跡されない組織) を実行するかどうかを指定します。",
```

Comment:
Same question here - `force:source:deploy` is now replaced with `project:deploy:start`.

Reply:
This PR is only for push commands, not deploy commands